doc/rados/operations/erasure-code-clay.rst

```diff
@@ -60,6 +60,7 @@ To create a new clay code profile::
              [scalar_mds={plugin-name}] \
              [technique={technique-name}] \
              [crush-failure-domain={bucket-type}] \
+             [crush-device-class={device-class}] \
```

Comment:
@runsisi i think this option is optional, right? if yes, could you please amend the commit message that this change is made for the sake of completeness? IMHO, "fix" in title would imply a bug.

Reply:
@tchaikov updated, please take another look